minio/known_errors.py

```diff
@@ -0,0 +1,434 @@
+# Author: MrRedAmber (https://github.com/MrRedAmber)
```

Comment:
Explicitly authors are not added as per our policy, we need to retain `Apache` license header from other file with date 2017.